src/utils/daos_hdlr.c

```diff
@@ -42,6 +43,10 @@
 
 #define OID_ARR_SIZE 8
 
+#if D_HAS_WARNING(4, "-Wframe-larger-than=")
```

Comment:
im not sure if this is going to be acceptable. probably better to allocate dynamically

Reply:
I found several files containing "#if D_HAS_WARNING(4, "-Wframe-larger-than=")", so I thought it would be fine.

Reply:
They should all be in test code but I'll take a look to verify.

Reply:
Mohamad is correct, can you remove this and allocate the buffer dynamically please.

Reply:
Fixed. D_ALLOC() is used.